src/components/util.tsx

```diff
@@ -1,8 +1,15 @@
-import { useTransition, type HTMLAttributes } from "react"
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
Either remove or add justification for disabling.

Reply:
More disability stuff